src/ui/layout/qgslayoutlegendwidgetbase.ui

```diff
@@ -273,7 +273,7 @@
                <string>Add group</string>
               </property>
               <property name="text">
-               <string>…</string>
+               <string>..</string>
```

Comment:
This change is certainly not on purpose

Reply:
no, just a weird side effect of using the widget builder.